restricting-traffic/default-deny/default-deny-nodelocaldns.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
Ok, so that would break the [tutorial I _just_ published](https://cloud.google.com/solutions/managing-cloud-infrastructure-using-kpt#apply_the_anthos_security_blueprint).
Can you instead create a folder `default-deny-nodelocaldns` and put what's related to that in there?
The problem is that kpt downloads an entire folder at once.

Reply:
done.